src/App.tsx

```diff
@@ -1,75 +1,121 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
+import 'bulma/css/bulma.min.css';
 import React from 'react';
-import './App.css';
-
-const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
-
-enum SortType {
-  NONE,
-  ALPABET,
-  LENGTH,
+import { Button } from 'react-bulma-components';
+import './App.scss';
+import { GoodList, Product } from './components/GoodList';
+import { goodsFromServer } from './data/goodsFromServer';
+
+type Props = {};
+
+interface State {
+  isStarted: boolean,
+  goods: Product[],
 }
 
-// Use this function in the render method
-function getReorderedGoods(
-  goods: string[],
-  sortType: SortType,
-  isReversed: boolean,
-) {
-  // Not to mutate the original array
-  const visibleGoods = [...goods];
+export class App extends React.Component<Props, State> {
+  state: State = {
+    isStarted: false,
+    goods: goodsFromServer,
```

Comment:
you've deleted this comment, but I'm here for you) don't save goods to state, they'll not change anyways
![image](https://user-images.githubusercontent.com/37161315/179500199-51111fd0-71b8-48e1-b95c-77f8881d97ba.png)